src/openeo_grass_gis_driver/actinia_processing/config.py

```diff
@@ -25,3 +31,51 @@ class Config:
     # The database file that stores the actinia jobs
     ACTINIA_JOB_DB = "%s/.actinia_job_db_file.sqlite" % os.environ["HOME"]
     SECRET_KEY = "jaNguzeef4seiv5shahchimoo8teiLah"
+
+
+class Configfile:
+
+    def __init__(self):
+        """
+        This class will overwrite the config classes above when config files
+        named DEFAULT_CONFIG_PATH/**/*.ini exist.
+        On first import of the module it is initialized.
+        """
+
+        config = configparser.ConfigParser()
+        config.read(CONFIG_FILES)
+
+        if len(config) <= 1:
+            print("Could not find any config file, using default values.")
+            return
+        print("Loading config files: " + str(CONFIG_FILES) + " ...")
```

Comment:
Move up before `config.read(CONFIG_FILES)` ?

Reply:
Just for clarification, I meant moving up only the `print(...)` line because now it comes after reading the config files, a bit late maybe.